client/verta/docs/tutorials/packaging/version_from_artifacts.rst

```diff
@@ -0,0 +1,53 @@
+Creating model versions from artifacts
+======================================
+
+Once `a registered model has been created <create_registered_model.html>`_, we can add new versions to it. With model versions, we can store necessary information for deployment, such as artifacts, model, requirements, etc.
+
+This tutorial will explain how we can create a version from artifacts, using the client and CLI.
+
+Using the client
+----------------
+
+Suppose we have just trained the following LogisticRegression classifier from scikit-learn:
+
+.. code-block:: python
+
+    classifier = LogisticRegression()
+    classifier.fit(X_train, y_train)
+
+We can create a new version for the registered model, and upload the classifier, along with the requirements (scikit-learn), as follows:
+
+.. code-block:: python
+
+    model_version = registered_model.create_version(
+        name="my version",
+        labels=["prototype"],
+    )
+
+    # Logging the classifier and requirements:
+    model_version.log_model(classifier)
+
+    reqs = Python.read_pip_file(requirements_path)
+    model_version.log_environment(Python(requirements=reqs))
+    model_version.log_environments(Python(requirements=["scikit-learn"]))
```

Comment:
Is this allowed? What happens in such case? Don't we have to enable overwrite?

Reply:
Oh, this is _not_ allowed. And `log_environment()` doesn't have an `overwrite`